core/src/modules/ops/recordProperty.ts

```diff
@@ -47,7 +45,7 @@ export namespace RecordPropertyOps {
           rawValue = await formatEmail(stringifiedValue);
           break;
         case "phoneNumber":
-          rawValue = await formatPhoneNumber(stringifiedValue);
+          rawValue = await formatString(stringifiedValue);
```

Comment:
Above this line we are still doing `    const stringifiedValue = \`${value}\`.trim();`. I though we also wanted to stop trimming strings in this PR?

Reply:
Oh, missed that. It didn't seem to affect the tests so I'm going to add one.

Reply:
email validation fails if it ends in a space. Is that desired behavior?

Reply:
I don't think so... I think we said that /whatever/ the user has we try to use so ` someone@fake .com ` - spaces all over the place - we would store as-is.

Reply:
I just want to be very intentful with any change here. Right now we do a check called `isEmail`
```
export default function isEmail(string: string) {
  if (string.match(/^\S+@\S+\.\S+$/)) return true;
  return false;
}
```
This fails if the email has leading or trailing spaces. We could scope this down to `@\S+\.` which just looks for an `@` and a `.`.
Double checking the ticket it looks like we want no validation on phone numbers, but less massaging of the emails.
I feel like if someone chooses `type: email` they'd expect at least some sort of validation compared with string, however I'm not wedded to this view.

I personally feel like this current version, where leading/trailing spaces is an error seems okay. But happy to go either way.

Reply:
I think that's a call for @bleonard!